bpm/src/main/java/org/jboss/pnc/bpm/task/AnalyzeDeliverablesTask.java

```diff
@@ -0,0 +1,48 @@
+package org.jboss.pnc.bpm.task;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import lombok.ToString;
+import org.jboss.pnc.api.dto.Request;
+import org.jboss.pnc.bpm.BpmTask;
+import org.jboss.pnc.bpm.model.AnalyzeDeliverablesBpmRequest;
+import org.jboss.pnc.spi.exception.CoreException;
+
+import java.io.Serializable;
+import java.util.HashMap;
+
+@ToString(callSuper = true)
+public class AnalyzeDeliverablesTask extends BpmTask {
```

Comment:
What do we get by extending BpmTask for a stateless approach and only RestConnector support ?

Reply:
The only thing I needed from BpmTask.class is the method getExtendedProcessParamameters() which returns the actual parameters.

https://github.com/project-ncl/pnc/blob/e05d714bd0df726ad579ad19c4073ec5eb501c22/bpm/src/main/java/org/jboss/pnc/bpm/BpmTask.java#L168-L200

Reply:
Now the task doesn't use BpmTask. We get processParameters through direct json serialization.